library/SaveScreenshotsToS3/script.js

```diff
@@ -1,36 +1,21 @@
-// Theshold for duration of entire script - fails test if script lasts longer than X (in ms)
-// Default is '0', which means that it won't fail.
-var ScriptTimeout = 60000;
-// Script-wide timeout for all wait and waitAndFind functions (in ms)
-var DefaultTimeout = 20000;
+// Script-wide timeout for all wait functions (in ms)
+var elementWaitTimer = 10000;
 
 // Change to any User Agent you want to use.
 // Leave as "default" or empty to use the Synthetics default.
 //var UserAgent = "default";
 var UserAgent = "default";
 
-const fs = require('fs');
-var req = require('request'),
-  By = $driver.By,
+const AWS = require('aws-sdk'),
   startTime = Date.now();
 
-//Start a transaction
-var startTrnx = function (thisStep) {
-  startTime = Date.now();
-  console.log(thisStep + ' Started.');
-};
-
-//End a transaction, log the time to Insights
-var endTrnx = function (thisStep) {
-  var lastStepTimeElapsed = Date.now() - startTime;
-  console.log(thisStep + ' Finished: ' + lastStepTimeElapsed + 'ms.');
-  $util.insights.set(thisStep, lastStepTimeElapsed);
-};
-
 var curDate = timestamp(); //UTC date for current execution of script (this is included in the folder name created within your bucket)
-var jobId = $env.JOB_ID //Monitor execution ID generated during run
+var jobId = $env.JOB_ID
+
+const url_to_monitor = 'https://www.tutorialrepublic.com/snippets/preview.php?topic=bootstrap&file=simple-login-form';
```

Comment:
Please remove url

Reply:
It's part of the script template, no need to remove